src/ensembl/src/content/app/entity-viewer/gene-view/GeneView.tsx

```diff
@@ -4,10 +4,12 @@ import { useQuery } from '@apollo/react-hooks';
 import { gql } from 'apollo-boost';
 
 import { getEntityViewerActiveEnsObject } from 'src/content/app/entity-viewer/state/general/entityViewerGeneralSelectors';
-
 import GeneOverviewImage from './components/gene-overview-image/GeneOverviewImage';
-
 import DefaultTranscriptslist from './components/default-transcripts-list/DefaultTranscriptsList';
+import GeneViewTabs from './components/gene-view-tabs/GeneViewTabs';
+import { getEntityViewerActiveGeneTab } from 'src/content/app/entity-viewer/state/gene-view/entityViewerGeneSelectors';
```

Comment:
_stylistic:_ move ⬆️  to selector imports?

Reply:
That's done